lua/nvim-treesitter/utils.lua

```diff
@@ -19,7 +19,7 @@ end
 
 function M.get_package_path()
   for _, path in pairs(api.nvim_list_runtime_paths()) do
-    if string.match(path, '.*/nvim%-treesitter') then
+    if string.match(path, '.*/nvim%-treesitter') or string.match(path, '.*\\nvim%-treesitter') then
```

Comment:
Can we use `fnamemodify` here instead ? That would avoid us some headaches

Reply:
Maybe fnamemodifiy brings new headaches? You can check whether the new version is better.